app.py

```diff
@@ -55,16 +56,25 @@ def populate_cfg_by_env(cfg):
     env_model_settings['access_key_id'] = os.environ.get('S3_ACCESS_KEY')
     env_model_settings['secret_access_key'] = os.environ.get('S3_SECRET_KEY')
 
+    env_common_settings = {}
+    env_common_settings['credentials'] = \
+        json.loads(os.environ.get('SERVICE_CREDENTIALS'))
```

Comment:
(It is more about the commit message) I doubt we should suggest to store secrets in the config file. Environment is okay. A separate file would be okay too (however I don't see a need, to be honest).

I think env is enough.

Before this commit all environment variables (and they're secrets in fact) were in the model settings. This one is in common settings. Why?

Reply:
> (It is more about the commit message) I doubt we should suggest to store secrets in the config file. Environment is okay. A separate file would be okay too (however I don't see a need, to be honest).
> 
> I think env is enough.

All secrets will be stored in environment variables (at least on this stage).

> 
> Before this commit all environment variables (and they're secrets in fact) were in the model settings. This one is in common settings. Why?

Because this variable doesn't have any relation to the model. I decided not add yet another special group and added `common`.

Reply:
Okay.